lists-service/src/main/resources/application.properties

```diff
@@ -2,7 +2,7 @@ app.name=lists-api
 app.version=1.0.0
 app.description=${app.name} is a Spring Boot application
 app.url=https://lists.test.ala.org.au
-app.fallbackUrl=
+# app.fallbackUrl=
```

Comment:
So devs can have ext config set - precedence means this has to be unset to work. Not used in server deployments, so safe to change.